src/contracts/test/ERC20Mintable.sol

```diff
@@ -1,14 +0,0 @@
-pragma solidity ^0.6.12;
```

Comment:
This should have been removed in #37 but stayed around by accident because of a rebase mistake :see_no_evil:

Reply:
Awesome! So we use the OZ mintable token now from our dependencies instead!